helper/populate_db.py

```diff
@@ -1,62 +1,76 @@
 import csv
 import logging
 import os
-from datetime import datetime, date, timedelta
-from dateutil import tz
-
-from dbschema.models import (DialogClosedAnswers, DialogOpenAnswers, DialogQuestions, Users,
-                             UserInterventionState,
-                             FirstAidKit, InterventionActivity, InterventionComponents,
-                             InterventionPhases,
-                             ClosedAnswers, InterventionActivitiesPerformed, Testimonials,
-                             StepCounts, UserStateMachine)
+
+from typing import Any
+from dbschema.models import (DialogQuestions, InterventionActivity, InterventionComponents,
+                             InterventionPhases, ClosedAnswers, Testimonials)
 from helper.helper_functions import get_db_session
 from helper.definitions import (Phases, Components, ComponentsTriggers,
                                 DialogQuestionsEnum, Notifications, NotificationsTriggers)
 
-tz_nl = tz.gettz("Europe/Amsterdam")
-
 
-def populate_db_with_test_data(session, test_user_id,
-                               activities_file_path='../utils/activities.csv',
-                               testimonials_file_path='../utils/testimonials_with_user_data.csv'):
+def populate_db_fixed_data(session,
+                           activities_file_path='../utils/activities.csv',
+                           testimonials_file_path='../utils/testimonials_with_user_data.csv'):
     """
     Populate the database with test data. Update data if it already exists.
```

Comment:
Should this still say "test data"? It seems like these tables are always needed even in prod

Reply:
Or will something else ensure that all the right tables are present in non-testing runs?

Reply:
My laziness doesn't let me read the descriptions ahah! I will update it. There is a function for checking that the tables are populated.